nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -182,6 +177,7 @@ public class Game : MonoSingleton<Game>
         protected override void Awake()
         {
             Debug.Log("[Game] Awake() invoked");
+            GL.Clear(true, true, Color.black);
```

Comment:
Was it the intended change?

Reply:
I've just moved it from Start(). It fits better in Awake().

Reply:
🆗